src/js/component.js

```diff
@@ -369,6 +369,10 @@ class Component {
       // If there's no .player_, this is a player
       let ComponentClass = Component.getComponent(componentClassName);
 
+      if (!ComponentClass) {
```

Comment:
Children could also, theoretically, be configured dynamically, in which case, we possibly don't want to throw.
However, now that I'm making sure that everything is a component below, I guess I probably don't even need to have this section here and leave it as what it was.